package.json

```diff
@@ -34,25 +34,25 @@
   },
   "homepage": "https://github.com/ProjectLibertyLabs/frequency-scenario-template#readme",
   "dependencies": {
-    "@dsnp/frequency-schemas": "^1.1.0",
-    "@dsnp/graph-sdk": "^1.1.3",
-    "@frequency-chain/api-augment": "1.13.1",
-    "@polkadot/api": "^13.0.1",
+    "@dsnp/graph-sdk": "^1.1.4",
+    "@frequency-chain/api-augment": "1.13.2",
+    "@polkadot/api": "^13.2.1",
     "@polkadot/keyring": "^13.1.1",
-    "@polkadot/types": "^13.0.1",
-    "@polkadot/types-codec": "^13.0.1",
+    "@polkadot/types": "^13.2.1",
+    "@polkadot/types-codec": "^13.2.1",
     "@polkadot/util": "^13.1.1",
     "@polkadot/util-crypto": "^13.1.1",
     "@types/minimist": "^1.2.5",
-    "@types/node": "^20.14.2",
+    "@types/node": "^22.7.4",
     "avsc": "^5.7.7",
     "loglevel": "^1.9.1",
     "minimist": "^1.2.8",
     "rxjs": "^7.8.1",
     "unique-names-generator": "^4.7.1"
   },
   "devDependencies": {
-    "@polkadot/typegen": "^13.0.1",
+    "@dsnp/schemas": "^1.3.2",
```

Comment:
is it a dev dependency here?

Reply:
It is now. It is only used in examples, so I consider that a devDependency.